kubernetes.go

```diff
@@ -44,32 +62,71 @@ type Kubernetes struct {
 	PersistentVolumeClaims *persistentvolumeclaims.PersistentVolumeClaims
 }
 
-type KubernetesOptions struct {
+// KubeConfig represents the initialization settings for the kubernetes api client.
+type KubeConfig struct {
 	ConfigPath string
 }
 
-func (obj *Kubernetes) XKubernetes(ctx *context.Context, options KubernetesOptions) (*Kubernetes, error) {
+// Ensure the interfaces are implemented correctly.
+var (
+	_ modules.Module   = &RootModule{}
+	_ modules.Instance = &ModuleInstance{}
+)
+
+// NewModuleInstance implements the modules.Module interface to return
+// a new instance for each VU.
+func (*RootModule) NewModuleInstance(vu modules.VU) modules.Instance {
+	mi := &ModuleInstance{
+		Version: version,
```

Comment:
If the idea is that this needs to  be exported - it needs to be in the exports

Reply:
Previous author brought in versioning in the module, so I just kept it, but will definitely add to the exports so that it may be echoed from the javascript.